src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Research what this is doing for you.

Reply:
Slightly researched looks to be testing for jest i believe was just in the create react default files. The whole file could probably be deleted based on the testing libraries we plan on using.